.gitignore

```diff
@@ -107,6 +107,11 @@ unlinked_spec.ds
 **/macos/Flutter/ephemeral
 **/xcuserdata/
 
+# Windows
+**/windows/flutter/generated_plugins.cmake
```

Comment:
Turns out this should be checked-in. Apologies for the churn! See: https://github.com/flutter/flutter/pull/110114#discussion_r955500880

```suggestion
```

Reply:
No problem, thanks for the clarification!